### ISSUE TITLE
why needs torch.distributions.utils.probs_to_logits?

### ISSUE THREAD

**Issue by tangyipeng100** [aRI0U/RandLA-Net-pytorch#26]
I have noticed that there comes  torch.distributions.utils.probs_to_logits after forward propagation:

train.py 115~117:
```
scores = model(points)#[b, 40960, 6]->[b, 14, 40960]

logp = torch.distributions.utils.probs_to_logits(scores, is_binary=False)
```
why needs torch.distributions.utils.probs_to_logits? Can I remove it and use _scores_ directly to calculate the loss?
